wikipedia-dark.user.css

```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.5.7
+@version      3.5.8
```

Comment:
Please dont add revision bumps with PR's, this will cause conflicts sooner rather than later,

Reply:
Reverted. Sorry about that